src/reimer_arenkiel_lab_to_nwb/embargo2024/embargo2024_convert_session.py

```diff
@@ -2,86 +2,105 @@
 
 from pathlib import Path
 from typing import Union
-from zoneinfo import ZoneInfo
-from natsort import natsorted
-
+from time import time
+from tqdm import tqdm
 from neuroconv.utils import load_dict_from_file, dict_deep_update
+from neuroconv.tools.nwb_helpers import configure_and_write_nwbfile
 
 from reimer_arenkiel_lab_to_nwb.embargo2024 import Embargo2024NWBConverter
-
+from reimer_arenkiel_lab_to_nwb.dj_utils import *
```

Comment:
I would make this import explicitly show what functions are imported from dj_utils